pom.xml

```diff
@@ -27,8 +27,7 @@
         <xml-format-maven-plugin.version>3.3.1</xml-format-maven-plugin.version>
         <checkstyle.version>10.18.1</checkstyle.version>
         <jsoup.version>1.18.1</jsoup.version>
-        <!-- TODO: uncomment Camel Quarkus BOM when Camel Quarkus 3 is released -->
```

Comment:
Shouldn't this be removed? I'm not sure if we have any camel related test coverage (anymore).
And the commit is not enabling any.

Reply:
Since @michalvavrik was the last person to change this in https://github.com/quarkus-qe/quarkus-test-suite/pull/1060 , this is probably a question for him

Reply:
I'd definitely remove it. I think removed Kamelet module and didn't do it. Let's drop it.

Reply:
Removed